featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -942,3 +970,7 @@ def strip_values_if_series(values):
     if isinstance(values, pd.Series):
         values = values.values
     return values
+
+
+def has_agg_type_in_primitive_or_feature(feature):
```

Comment:
Maybe we don't need this function, since it's just 1 line

Reply:
@gsheni 

Yeah, I am still trying to play around with some ideas; I'm not sure the one I have up right now will end up working. 

Just to scope out the goals for this PR: 

-- Add series_type param (default to Pandas) to transform primitives 
-- Add series_type param (default to Pandas) to aggregation primitives
-- Handle case where user passes in "agg_type" as keyword argument instead of "series_type"

Does that sound about right?

Reply:
Yes, the last one is only for aggregationPrimtives

Reply:
I think we can make `agg_type` a keyword-only argument. This would allow backwards compatibility for users who specified `agg_type` in the call to `get_function`, and automatically handle it if they didn't. 
https://python-3-for-scientists.readthedocs.io/en/latest/python3_advanced.html

Reply:
Sounds good!